examples/examples/balance_transfer_status_stream.rs

```diff
@@ -0,0 +1,54 @@
+use futures::StreamExt;
+use sp_keyring::AccountKeyring;
+use subxt::{
+    tx::{PairSigner, TxStatus},
+    OnlineClient, PolkadotConfig,
+};
+
+// Generate an interface that we can use from the node's metadata.
+#[subxt::subxt(runtime_metadata_path = "../artifacts/polkadot_metadata.scale")]
+pub mod polkadot {}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let api = OnlineClient::<PolkadotConfig>::new().await?;
+
+    let dest = AccountKeyring::Bob.to_account_id().into();
```

Comment:
I would include the same introductory code comments above this, and the `api` variable, etc that you do in `balance_transfer_basic.rs` assuming someone comes into this code example without seeing other ones, they should have the same level of explanation.